.travis.yml

```diff
@@ -85,7 +85,8 @@ before_script:
 script:
   - vendor/bin/codecept build --quiet
   - vendor/bin/codecept run --ext DotReporter tests/cli/
-  - vendor/bin/codecept run --ext DotReporter tests/integration/
+#  - vendor/bin/codecept run --ext DotReporter tests/integration/
+  - vendor/bin/codecept run  tests/integration/
```

Comment:
Can we put back DotReporter?

Reply:
Leave it for now, I will revert it in the next PR.